modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/trigger/messaging/ObjectActionTriggerMessageListener.java

```diff
@@ -37,9 +39,12 @@ public ObjectActionTriggerMessageListener(
 
 	@Override
 	protected void doReceive(Message message) {
+		long companyId = GetterUtil.getLong(message.get("companyId"));
+
 		_objectActionEngine.executeObjectActions(
-			_className, GetterUtil.getLong(message.get("companyId")),
-			_objectActionTriggerKey, (JSONObject)message.getPayload(),
+			StringBundler.concat(_className, StringPool.POUND, companyId),
```

Comment:
This change is to concat _systemObject_ _className_ with _companyId_. I tested and only system object pass for this method. If _customObject_ pass here the result is: `className#objectDefinitionId#companyId` and will not work. I can add a validation for customObject, what do you think?

Reply:
@murilostodolni I think it would be good to add this validation

Reply:
As discussion with @carolmariaabb, this class is used only to commerce custom action: On Order Status Update, On Payment Status Update e On Subscription Status Update. Because of this, we don't need add validation for customObject in this place.